reddit.js

```diff
@@ -1,5 +1,48 @@
 var reddit = require("./data.json")
 
-// Write your solutions below
 
-console.log(reddit["data"]);
\ No newline at end of file
+
+// Question 1: Use foreach loop to console.log all titles 
+
+reddit.data.children.forEach(function(childrenObject) {console.log(childrenObject.data.title)} );
```

Comment:
Looks great! Let's just format this so that it follows convention:
```
reddit.data.children.forEach(function(childrenObject){
    console.log(childrenObject.data.title)
});
```